src/cli.rs

```diff
@@ -18,38 +18,50 @@ mod generate_config_schema;
 mod manage;
 mod run;
 
-use std::path::PathBuf;
+use std::{
+    path::{Path, PathBuf},
+    sync::Arc,
+};
+
+use tokio::{signal, sync::watch};
 
 use crate::Config;
 
+pub use self::{
+    generate_config_schema::GenerateConfigSchema,
+    manage::{Manage, Providers},
+    run::Run,
+};
+
 const VERSION: &str = env!("CARGO_PKG_VERSION");
+const ETC_CONFIG_PATH: &str = "/etc/quilkin/quilkin.yaml";
 
+/// The Command-Line Interface for Quilkin.
 #[derive(clap::Parser)]
+#[non_exhaustive]
 pub struct Cli {
-    #[clap(
-        short,
-        long,
-        env = "QUILKIN_CONFIG",
-        default_value = "quilkin.yaml",
-        help = "The YAML configuration file."
-    )]
-    config: PathBuf,
-    #[clap(
-        short,
-        long,
-        env,
-        help = "Whether Quilkin will report any results to stdout/stderr."
-    )]
-    quiet: bool,
+    /// The path to the configuration file for the Quilkin instance.
+    #[clap(short, long, env = "QUILKIN_CONFIG", default_value = "quilkin.yaml")]
```

Comment:
I was actually thinking of coming back around and fleshing out the help for the rest of the commands for a nice command line reference.

Curious if you didn't want a command line reference here?

Reply:
Oh wait - when I run the command, I see help everywhere, 

Wait.... does clap take the help from the comment above it? If so, that's a super neat trick.